README.md

```diff
@@ -4,24 +4,25 @@
 <a href="https://github.com/camaraproject/§repo_name§/graphs/contributors" title="Contributors"><img src="https://img.shields.io/github/contributors/camaraproject/§repo_name§?style=plastic"></a>
 <a href="https://github.com/camaraproject/§repo_name§" title="Repo Size"><img src="https://img.shields.io/github/repo-size/camaraproject/§repo_name§?style=plastic"></a>
 <a href="https://github.com/camaraproject/§repo_name§/blob/main/LICENSE" title="License"><img src="https://img.shields.io/badge/License-Apache%202.0-green.svg?style=plastic"></a>
+<a href="https://github.com/camaraproject/§repo_name§/releases/latest" title="Latest Release"><img src="https://img.shields.io/github/release/camaraproject/§repo_name§?style=plastic"></a>
 
 # §repo_name§
-Repository to describe, develop, document and test the §repo_name§ API family
+Repository to describe, develop, document, and test the §repo_name§ API family
 
 ## Scope
 * Service APIs for “§repo_name§” (see APIBacklog.md)  
 * It provides the customer with the ability to:  
   * §API family description§.
-  * NOTE: The scope of this API family should be limited (at least at a first stage) to 4G and 5G.  
-* Describe, develop, document and test the APIs (with 1-2 Telcos)  
+  * NOTE: The scope of this API family should be limited (at least at the first stage) to 4G and 5G.  
+* Describe, develop, document, and test the APIs (with 1-2 Telcos)  
 * Started: §start date§
-* Location: virtually  
+* [Latest Release](https://github.com/camaraproject/§repo_name§/releases/latest)
 
-## Meetings
-* Meetings are held virtually
-* Schedule: tbd
-* Meeting link: tbd
-
-## Contributorship and mailing list
-* To subscribe / unsubscribe to the mailing list of this Sub Project and thus be / resign as Contributor please visit <https://lists.camaraproject.org/g/sp-§api_family_abbreviation§>.
-* A message to all Contributors of this Sub Project can be sent using <sp-§api_family_abbreviation§@lists.camaraproject.org>.
+## Contributing
```

Comment:
I agree here with @MarkusKuemmerle and propose the following section (don't know if the comments would work as expected within README.md, but worth a try):

```suggestion
## Status and released versions
<!-- Use/uncomment one or multiple the following options -->
<!-- The sub project has no (pre)releases yet, work in progress is within the main branch -->
<!-- Pre-releases of this sub project are available in https://github.com/camaraproject/§repo_name§/releases -->
<!-- The latest public release is available here: https://github.com/camaraproject/§repo_name§/releases/latest -->
<!-- For changes see [CHANGELOG.md](https://github.com/camaraproject/§repo_name§/blob/main/CHANGELOG.md) -->

## Contributing
```